pennylane_forest/qpu.py

```diff
@@ -148,21 +127,15 @@ def __init__(
             """dict[str, pyquil.quilatom.MemoryReference]: stores the string of symbolic
                 parameters associated with their PyQuil memory references."""
 
-        timeout = kwargs.pop("timeout", None)
+        timeout = kwargs.pop("timeout", 10.0) # 10.0 is the pyquil default
```

Comment:
This is bound to drift into inaccuracy. Can't we just let pyQuil enforce its own defaults by not passing a `compiler_timeout` value at all if it's not specified?

Reply:
For most of the calls, yes. The exception would be [this pyQuil private method call](https://github.com/MarquessV/pennylane-forest/pull/1/files#diff-e54edc4229030c98c5b70918872608e43667b52ebfafa4f6c3d93996eea94466R111) which I happened to tackle first. The timeouts are required there, so we'd have to provide some value anyways. No reason we couldn't handle that specifically for that method and keep the other ones generic.

Between requiring a value for timeouts and a `QCSClientConfiguration`, it probably makes sense to get rid of this private method call.